data/gpx_test_data/track_with_timestamps_broken.gpx

```diff
@@ -0,0 +1,28 @@
+<?xml version='1.0' encoding='UTF-8' standalone='yes' ?>
+<gpx version="1.0">
+<trk>
+    <name>new</name>
```

Comment:
nit: I wonder why 4 spaces are used everywhere instead of 2. It wastes a lot of space and slows down the parser...